test/template-create/out/my-template/v0/manifest.jsonnet

```diff
@@ -8,5 +8,121 @@
     variablesConfig: "variables",
     variablesValuesRow: "values/{config_row_id}",
   },
-  configurations: [],
+  configurations: [
+    {
+      componentId: "ex-generic-v2",
+      id: "empty",
+      path: "extractor/ex-generic-v2/empty",
+      rows: [],
+    },
+    {
+      componentId: "ex-generic-v2",
+      id: "without-rows",
+      path: "extractor/ex-generic-v2/without-rows",
+      rows: [],
+    },
+    {
+      componentId: "keboola.ex-db-mysql",
+      id: "with-rows",
+      path: "extractor/keboola.ex-db-mysql/with-rows",
+      rows: [
+        {
+          id: "disabled",
+          path: "rows/disabled",
+        },
+        {
+          id: "test-view",
+          path: "rows/test-view",
+        },
+        {
+          id: "users",
+          path: "rows/users",
+        },
+      ],
+    },
+    {
+      componentId: "keboola.shared-code",
+      id: "shared-codes",
+      path: "other/keboola.shared-code/shared-codes",
+      rows: [
+        {
+          id: "code-with-variables",
+          path: "codes/code-with-variables",
+        },
+        {
+          id: "my-code-1",
+          path: "codes/my-code-1",
+        },
+        {
+          id: "my-code-2",
+          path: "codes/my-code-2",
+        },
+      ],
+    },
+    {
+      componentId: "keboola.variables",
+      id: "shared-code-variables",
+      path: "variables",
+      relations: [
+        {
+          configId: "shared-codes",
+          rowId: "code-with-variables",
+          type: "sharedCodeVariablesFor",
+        },
+      ],
+      rows: [],
+    },
+    {
+      componentId: "transformation",
+      id: "old-transformation",
+      path: "other/transformation/old-transformation",
+      rows: [
+        {
+          id: "old-transformation-snfk",
+          path: "rows/old-transformation-snfk",
+        },
+      ],
+    },
+    {
+      componentId: "keboola.python-transformation-v2",
+      id: "python-transformation",
+      path: "transformation/keboola.python-transformation-v2/python-transformation",
+      rows: [],
+    },
+    {
+      componentId: "keboola.python-transformation-v2",
+      id: "transformation-with-shared-code",
+      path: "transformation/keboola.python-transformation-v2/transformation-with-shared-code",
+      rows: [],
+    },
+    {
+      componentId: "keboola.variables",
+      id: "transformation-with-shared-code-variables",
+      path: "variables",
+      relations: [
+        {
+          componentId: "keboola.python-transformation-v2",
+          configId: "transformation-with-shared-code",
+          type: "variablesFor",
+        },
```

Comment:
IDs su nahradene na vsetkych urovniach.
Teda to bude fungovat aj do buducna, ... lubovolny mapper, ktory niekde ulozi `BranchId/ConfigId/RowId` (alebo `Key`) bude automaticky fungovat aj s templates.

Reply:
Jedine `defaultbucket` mapper zatial nepojde, lebo ten si do internal state uklada cestu nie ID - to sa fixne samostatne.